doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -198,6 +198,35 @@ system.
 | `id`                | [`text`]         | The ID of the source. Corresponds to [`mz_catalog.mz_sources.id`](../mz_catalog#mz_sources).                   |
 | `replication_slot`  | [`text`]         | The name of the replication slot in the PostgreSQL database that Materialize will create and stream data from. |
 
+<!-- ### `mz_prepared_statement_history` -->
```

Comment:
Nit: Are multi-line comments possible? If so I suggest putting all of this into a single comment, for a much better diff when you make this a non-comment later.